README.md

```diff
@@ -2,7 +2,7 @@
 
 > Cyberpunk HyperTerm theme
 
-![](https://cdn.rawgit.com/staltz/hyperpunk/master/screenshot.png)
+![](screenshot.png)
```

Comment:
I think we should be safe here... if you check out another theme I made:

  * https://github.com/brianegan/hyperterm-finest-base16-ocean-dark 

And the corresponding npm page for it: 

  * https://www.npmjs.com/package/hyperterm-finest-base16-ocean-dark

The screenshot is showing up as expected. Does that work from your side?